dagrunner/execute_graph.py

```diff
@@ -235,7 +235,7 @@ def _get_networkx(networkx_graph):
     else:
         try:
             edges, nodes = networkx_graph
-            nodes = {k: nodes[k] | _process_nodes(k) for k in nodes.keys()}.items()
+            nodes = {k: _process_nodes(k) | nodes[k] for k in nodes.keys()}.items()
```

Comment:
This is the simple change of this PR... (changing the order of the precedence to allowing node attributes to override Node properties.